packages/calcite-components/src/components/list/usage/Advanced.md

```diff
@@ -1,37 +1,34 @@
 ```html
 <calcite-list>
-  <calcite-list-item label="Princess Bubblegum" description="Ruler of The Candy Kingdom">
+  <calcite-list-item
+    label="Dog"
+    description="(Canis familiaris) A carnivorous mammal closely related to the gray wolf that has been domesticated as a pet."
+  >
     <calcite-action icon="drag" label="drag" scale="s" slot="actions-start"></calcite-action>
-    <calcite-icon scale="l" icon="effects" slot="content-start"></calcite-icon>
-    <calcite-avatar
-      scale="l"
-      slot="content-start"
-      thumbnail="https://slm-assets.secondlife.com/assets/19947929/view_large/Capture_du_2018-03-04_20-40-56.jpg?1520192584"
-    ></calcite-avatar>
+    <calcite-icon scale="l" icon="layer" slot="content-start"></calcite-icon>
+    <calcite-avatar scale="l" slot="content-start" thumbnail="https://placedog.net/300/300"></calcite-avatar>
     <calcite-icon scale="s" icon="check" slot="content-end" style="color: var(--calcite-ui-success)"></calcite-icon>
     <calcite-action icon="ellipsis" label="menu" slot="actions-end"></calcite-action>
     <calcite-action icon="x" label="remove" slot="actions-end"></calcite-action>
   </calcite-list-item>
-  <calcite-list-item label="Finn Mertens" description="Part owner of the Tree House">
+  <calcite-list-item
+    label="Cat"
+    description="(Felis catus) A carnivorous mammal that has long been domesticated as a pet and for catching rats and mice."
+  >
     <calcite-action icon="drag" label="drag" scale="s" slot="actions-start"></calcite-action>
-    <calcite-icon scale="l" icon="running" slot="content-start"></calcite-icon>
-    <calcite-avatar
-      scale="l"
-      slot="content-start"
-      thumbnail="https://www.seekpng.com/png/detail/90-906849_89kib-1024x631-finn-finn-adventure-time-face.png"
-    ></calcite-avatar>
+    <calcite-icon scale="l" icon="layer" slot="content-start"></calcite-icon>
+    <calcite-avatar scale="l" slot="content-start" thumbnail="https://placekitten.com/g/300/300"></calcite-avatar>
     <calcite-icon scale="s" icon="check" slot="content-end" style="color: var(--calcite-ui-success)"></calcite-icon>
     <calcite-action icon="ellipsis" label="menu" slot="actions-end"></calcite-action>
     <calcite-action icon="x" label="remove" slot="actions-end"></calcite-action>
   </calcite-list-item>
-  <calcite-list-item label="Jake T. Dog" description="Part owner of the Tree House">
+  <calcite-list-item
+    label="Asteroid"
```

Comment:
Is this meant to be Asteroid, or possibly another animal? 🐻

Reply:
Good catch!